serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-grpc-services.adoc

```diff
@@ -1 +1,163 @@
-= Orchestration of gRPC based services
\ No newline at end of file
+= Orchestration of gRPC based services
+
+Learn how to invoke a gRPC from a Serverless Workflow running a quarkus. This guide covers:
+
+* Creating a workflow that invokes a running gRPC service. 
+* Setting up properties to locate the running service. 
+
+== Prerequisites
+
+It's expected you have read how to create-your-first-workflow-service.html[Create Your First Workflow Guide]. 
+
+It's assumed you are familiar with main https://grpc.io/docs/what-is-grpc/core-concepts/[gRPC] concepts. 
```

Comment:
```suggestion
It's assumed you are familiar with the main link:https://grpc.io/docs/what-is-grpc/core-concepts/[gRPC] concepts. 
```

Reply:
hmmm, I do not think the link: is needed